packages/functions/src/utils/DbAppointmentUtils.ts

```diff
@@ -120,6 +121,25 @@ export function toBookedAppointmentSync(
   }
 }
 
+export async function getAppointmentById(
```

Comment:
These 2 functions should be in the Appointment DAL, not here

Reply:
Done